.changeset/metal-fireants-switch.md

```diff
@@ -0,0 +1,52 @@
+---
+"@spectrum-css/dialog": major
+"@spectrum-css/modal": patch
+"@spectrum-css/underlay": patch
+---
+
+Spectrum 2 Standard Dialog Migration
+
+This is the migration for both standard dialog. The Divider component is no longer supported in S2 dialogs. In addition, some new and updated tokens are in place to update dialog corner rounding, font treatments, spacing/padding, and maximum/minimum widths. There were several new elements implemented as well, including optional header and footer content, hero/cover images, checkbox and text-only options in the footer content area.
```

Comment:
Thanks for writing up this detailed change log. I did see one small typo here around "both".